dashboard/app/models/user_level.rb

```diff
@@ -97,6 +98,11 @@ def handle_unsubmit
     end
   end
 
+  # clear the per-request cache when user levels are saved or updated
+  def clear_user_level_cache
+    user.clear_user_level_cache unless user.nil
+  end
```

Comment:
This method seems to not exist.

Further, I don't understand why we need to clear the value of an instance variable whose lifetime is that of the request.